src/m_misc.c

```diff
@@ -661,3 +661,36 @@ char *M_OEMToUTF8(const char *oem)
 
 #endif
 
+//
+// M_NormalizeSlashes
+//
+// Remove trailing slashes, translate backslashes to slashes
+// The string to normalize is passed and returned in str
+//
+// killough 11/98: rewritten
+//
+// [STRIFE] - haleyjd 20110210: Borrowed from Eternity and adapted to respect
+// the DIR_SEPARATOR define used by Choco Doom. This routine originated in
+// BOOM.
+//
+void M_NormalizeSlashes(char *str)
```

Comment:
Could you please adapt this function to the coding style documented in HACKING.md?

Reply:
This appears to now be resolved.